autode/config.py

```diff
@@ -158,6 +158,12 @@ class _ConfigClass:
     # value before the entropy is calculated
     vib_freq_shift = Frequency(100, units='cm-1')
     # -------------------------------------------------------------------------
+    # Frequency scale factor, useful for DFT functions known to have a
+    # systematic error. This value must be between 0 and 1. For example,
```

Comment:
Maybe worth reflecting that (0, 1] in the comment to reflect the code behaviour

Reply:
absolutely